test/user.test.js

```diff
@@ -16,14 +18,21 @@ describe('User', function() {
   var invalidCredentials = {email: 'foo1@bar.com', password: 'invalid'};
   var incompleteCredentials = {password: 'bar1'};
 
+  var defaultApp;
+
   beforeEach(function() {
-    User = loopback.User.extend('user');
+    // FIXME: [rfeng] Remove loopback.User.app so that remote hooks don't go
+    // to the wrong app instance
+    defaultApp = loopback.User.app;
+    loopback.User.app = null;
+    User = loopback.User.extend('TestUser', {}, {http: {path: 'test-users'}});
```

Comment:
Yes, because the model in use is a sub-model of User. If we don't explicitly specify `http.path`, it will default to /testusers.